cmd/nknc/commands/root.go

```diff
@@ -0,0 +1,82 @@
+package commands
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"net"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/howeyc/gopass"
+	"github.com/nknorg/nkn/v2/config"
+	"github.com/spf13/cobra"
+)
+
+// Globals
+var (
+	WalletFile     string
```

Comment:
I suggest we make all these variables private unless they are needed outside package (which seems not?)

Reply:
done.